### ISSUE TITLE
[TG] Showing incorrect triples in reference table when no Reference score computed

### ISSUE THREAD

**Issue by albertmeronyo** [King-s-Knowledge-Graph-Lab/ProVe#31]
[Reported via Telegram] I looked at Q38 as its one of your examples on the wikiproject. why do the listed triples say they are for the united states?

<img width="878" alt="image" src="https://github.com/user-attachments/assets/24b2e84c-62b7-4ed2-aa93-20059b52a50f">

This seems to happen when users click on the reference table but ProVe does not have a Reference score computed yet for that item.

**Comment by dignityc** [comment]
Thank you for bringing this issue to my attention. The problem is related to the Wikidata API. ProVe currently uses two types of Wikidata APIs: the official Wikidata API and a third-party API (the qwikidata Python library). Gabriel used the qwikidata library in the original code, so some parts of ProVe still rely on it. Although I have updated many parts of the code to use the official Wikidata API, some lines remain unchanged. My ultimate goal is to transition all API usage to qwikidata for future sustainability. While qwikidata worked well before, it is now returning incorrect data, as in your example. For instance, for the 'Adam Douglas' item, it provided a Chinese name even though there was no English name data extracted.

I have resolved the issue found in the 'Italy' example and will replace the remaining qwikidata lines with the official Wikidata API for the other items.
